addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5044,6 +5044,24 @@ registry.Box = SnippetOptionWidget.extend({
 
 
 registry.layout_column = SnippetOptionWidget.extend({
+    /**
+     * @override
+     */
+    cleanForSave() {
+        // Remove the grid preview.
+        if (this.gridPreviewEl) {
+            this._removeGridPreview();
+        }
+    },
+    /**
+     * @override
+     */
+    cleanUI() {
+        // Remove the grid preview.
+        if (this.gridPreviewEl) {
+            this._removeGridPreview();
+        }
+    },
```

Comment:
Same: would remove the `if` + duplicated code (cleanUI should be called at the same time as cleanForSave anyway).